exonum/src/runtime/supervisor/mod.rs

```diff
@@ -124,6 +124,16 @@ impl Service for Supervisor {
 }
 
 impl Supervisor {
-    pub const BUILTIN_ID: InstanceId = 0;
+    pub const BUILTIN_ID: InstanceId = SUPERVISOR_SERVICE_ID;
```

Comment:
Supervisor asking for its service id from the runtime seems counter-intuitive for me, to be honest.